.travis.yml

```diff
@@ -0,0 +1,89 @@
+sudo: required
+dist: trusty
+
+branches:
+  except:
+  #- l10n_dev
+
+language: node_js
+node_js:
+  - "8"
+  
+addons:
+apt:
+  sources:
+    - google-chrome
+  packages:
+    - google-chrome-stable
```

Comment:
dont use chrome, use phantom js, if we just confirm the functionalities.

http://karma-runner.github.io/0.8/plus/Travis-CI.html

Reply:
As the tests are set up, they are using Chrome in headless mode so these are needed to run the tests.